docs/api/instruments/hp/hp6641a.rst

```diff
@@ -0,0 +1,12 @@
+###########################################################
+System Power Supply (664xA, 665xA, 667xA, 668xA, and 669xA)
```

Comment:
Better to call it HP 66xxA series system power supply or similar.

Reply:
As I understand it, 66xxA may include a power supply that is not part of the series (664xA, 665xA, 667xA, 668xA, and 669xA). That's why I copied it from the datasheet.